recipes/codecov/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "codecov" %}
+{% set version = "2.0.6" %}
+{% set sha256 = "999bf4d6f7e072433601798f3f291277a4ab519b245af2971cdeb247dbb21454" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+#  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/codecov/codecov-python/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
```

Comment:
Looks like you'll also need to include a `patches` key in `source` since the project includes `argparse` as a dependency, which is a py26 only thing.  

```yaml
source:
  fn: ...
  url: ...
  sha256: ...
  patches:
    - setup.py.patch
```

That patch should remove argparse from the setup.py `install_requires` key. This is a valid patch for that purpose and should be placed in a setup.py.patch file next to the meta.yaml

```diff
diff --git a/setup.py b/setup.py
index d32c116..4ec82df 100644
--- a/setup.py
+++ b/setup.py
@@ -27,6 +27,6 @@ setup(name='codecov',
       packages=['codecov'],
       include_package_data=True,
       zip_safe=True,
-      install_requires=["requests>=2.0.0", "coverage", "argparse"],
+      install_requires=["requests>=2.0.0", "coverage"],
       tests_require=["unittest2"],
       entry_points={'console_scripts': ['codecov=codecov:main']})
```